modules/openapi-generator/src/main/resources/python/python-experimental/model_utils.mustache

```diff
@@ -25,6 +25,7 @@ if six.PY3:
     # these are needed for when other modules import str and int from here
     str = str
     int = int
+    bytes = bytes
```

Comment:
Can you remove this and the bytes definition on line 37?

Reply:
@spacether I originall put it there for consistency with the rest of the definitions (str and int). What's the reasoning not to also include bytes (and wouldn't not including here break imports in other locations)?

Reply:
We aren't using this bytes definition yet though, right?